supernode/server/api/api.go

```diff
@@ -47,6 +47,9 @@ func newCategory(name, prefix string) *category {
 	apiCategories[name] = &category{
 		name:   name,
 		prefix: prefix,
+		handlerSpecs: []*HandlerSpec{
+			listHandler(name),
```

Comment:
why should we add a default list handler here?

Reply:
I can help up to get all the available APIs provided by supernode.